### PR TITLE
Skylance sprite and other quarg outfits

### PR DESCRIPTION
**Pull request by Brick63**
the skylance sprite is the work of several people and was received very well on the discord and was only recently finished by me 
~~while only the skylance is done if a general theme is decided upon the rest should follow quite easily~~
these remaining sprites are to give the quarg more than just question marks
to do
- [x] agree on a design
- [x] skylance
- [x] anti missile
- [x] shields
- [x] thrusters and turning
- [x] batteries
- [x] (possible) antimatter core
- [x] intrusion countermeasures

assets
[quarg outfits.zip](https://github.com/endless-sky/endless-sky/files/1679505/quarg.outfits.zip)

### CODE REVIEW

**Comment by PompanoZombie** [comment]
Looks very nice. I feel like it is something I would put on a Skylark. The theme goes pretty well with how the Skylark looks. Hope this gets merged!

**Comment by Lineth** [comment]
Looks awesome. And I agree that it fits the Skylark, which always seemed more 'Quarg' to me.

**Comment by Brick63** [comment]
if the design is approved then the rest of the outfits will follow as doing them all would be hard to adjust if changes were required

**Comment by endless-sky** [comment]
To me it sort of looks like the three pointy bits are just sort of floating in space instead of being connected to anything.

**Comment by tehhowch** [comment]
Relevant precedent for one->many: #1837
I agree that with one completed, the design theme should be checked before going willy-nilly to the rest of them.

There is the matter of "Are the relevant Quarg outfits even *complete*", meaning how much change is there still to be done regarding the given outfits & their attributes. If they are still subject to a lot of change, then they probably shouldn't be illustrated as sprite informs design and vice versa (awesome-looking outfits get better stats). With a single sprite illustrated, there is still a lot of flexibility in the application of its theme to the rest of the outfits.

**Comment by Amazinite** [comment]
>To me it sort of looks like the three pointy bits are just sort of floating in space instead of being connected to anything.

That was (I believe) a purposeful design choice, meant to show just how advanced the Quarg are that they have turrets with levitating parts.

**Comment by PompanoZombie** [comment]
With little illustration completed for the Quarg besides their ships, I think this is a good first step. Even if the new sprite isn't merged, I think it is definitely a good basis for upcoming sprites to go off of. For me the sprite looks pretty Quarg Like.

**Comment by KiLEdEnNis** [comment]
basically the question now is: is this general theme fine?

and most of us seem to think: yes

**Comment by Brick63** [comment]
just added a list of tasks to help keep this organized

File: data/quarg outfits.txt [hunk 1]
```diff
@@ -50,6 +51,7 @@ outfit "Quarg Skylance"
 		"shield damage" 50
 		"hull damage" 40
 		"ion damage" .1
+	description "The skylance is seemingly built from a smooth, black, granite-like material, although obviously it is actually made of a far more advanced and sophisticated substance."
```
Comment (Bladewood) [inline data/quarg outfits.txt]: What *I* was hoping to use as a good description for the skylance went like this:
`The immensely powerful beam weapons used by Quarg warships strike fear into the hearts of all who dare oppose them.`

File: data/quarg outfits.txt [hunk 2]
```diff
@@ -39,6 +39,7 @@ outfit "Quarg Skylance"
 	weapon
 		sprite "projectile/skylance"
 			"frame rate" 1
+		"hardpoint sprite" "hardpoint/skylance"
```
Comment (Bladewood) [inline data/quarg outfits.txt]: Note that we need to specify the `"hardpoint offset"` as well. 9 should be good.

File: copyright [hunk 3]
```diff
@@ -270,6 +270,11 @@ Copyright: Various
 License: public-domain
  Taken from unsplash.com, a collection of photographs that have been donated and
  placed in the public domain.
+ 
+ Files: images/outfit/skylance*
+Copyright: Evan Fluharty (Evanfluharty@gmail.com)
+License: CC-BY-SA-4.0
+Comment: Derived and completed from works by Maximilian Korber(Wrzlprnft), Fernando S. Sáez(Azure Nyoki), and originally drawn up by bladewood (info needed)(all from under the same license)
```
Comment (Bladewood) [inline copyright]: There should be spaces between the full names and the usernames (and I'm not sure if we should actually put the latter).
I don't think diacritics are allowed here, though this is a copyright file, not a data file, so I'm not certain.
My name is `Tommy Thach`.

**Comment by Bladewood** [comment]
>> To me it sort of looks like the three pointy bits are just sort of floating in space instead of being connected to anything.
>
> That was (I believe) a purposeful design choice, meant to show just how advanced the Quarg are that they have turrets with levitating parts.

Indeed, precisely the original point of the design.
Here were my original drafts for the skylance: [paper and pencil](https://user-images.githubusercontent.com/16506683/34893569-cbbb2030-f7ab-11e7-90c1-100839997513.jpg), [paint.net](https://user-images.githubusercontent.com/16506683/34893572-cd2a234e-f7ab-11e7-9412-0469d02e4d8e.jpg).
Then Wrzlprnft came along and turned it into [this](https://user-images.githubusercontent.com/16506683/34893574-d40222ac-f7ab-11e7-8dbf-00815c83c6c3.png). Then Azure came along with [this](https://user-images.githubusercontent.com/16506683/34893648-3052390c-f7ac-11e7-9655-97c0f7a8a371.png), and finally Brick finished it off with the result here.

File: data/quarg outfits.txt (hunk 2 above)
Comment (Brick63) [reply data/quarg outfits.txt]: ahh okay

File: data/quarg outfits.txt (hunk 1 above)
Comment (Brick63) [reply data/quarg outfits.txt]: i just used something you posted as a place holder till something better was made

File: data/quarg outfits.txt (hunk 2 above)
Comment (Brick63) [reply data/quarg outfits.txt]: fixed

**Comment by Brick63** [comment]
if this is going to be the quarg style then @Bladewood would have to draw up all the designs to keep them consistent

**Comment by accursedCursive** [comment]
I don't like the look of this as a laser; it's twiggy looking, yet Skylance is the heaviest and most powerful laser in the game.
But it's _perfect_ as an anti-missile sprite, and obviously Quarg need one of those.
The modelling is great, the lighting matches other ES sprites, the one thing I think should be different is the materials of the icosahedron and the crystal thing because they look like plastic.

I made my own Skylance before I saw this thread, by the way:
skylance
skylance

**Comment by KiLEdEnNis** [comment]
the Quarg are a very advanced alien species with technology humans wouldn't even understand, don't assume that such a Skylance sprite would be weak; it has levitating pieces, some sort of energy core, and a crystal pumping out a high-damage laser beam, which seems illogical to us but seems very alien-like

as for your version of the Skylance:
- model looks way too plain and boring
- model also lacks details
- its base is too simplistic
- it lacks colors
- I think you used smooth shading all over? looks bad on the base

**Comment by accursedCursive** [comment]
It has colours, they're subtle.
Anyway, the Skylance weighs 60 tonnes.
If a 60 tonne piece of Quarg technology is a few floating sticks, then what's their anti missile technology going to look like? A hovering strand of uncooked spaghetti?

**Comment by KiLEdEnNis** [comment]
> It has colours, they're subtle.

it's a big-ass yellow turret, so no

> Anyway, the Skylance weighs 60 tonnes.
If a 60 tonne piece of Quarg technology is a few floating sticks

you're assuming the materials and whatever tech is in it can't be heavy, while it can
for example, maybe they used compression technology for the core

> then what's their anti missile technology going to look like? A hovering strand of uncooked spaghetti?

GitHub is not really the place for jokes; there's a time and place for everything, but not now
~~(and it's not even remotely funny)~~

**Comment by Brick63** [comment]
@accursedCursive That Sprite is too bland and lacking details and I would suggest going back and improving on it as this PR is mostly about getting a design theme for the quarg

**Comment by Wrzlprnft** [comment]
Thumbs up for wording your first comment as a subjective opinion rather than a set fact, accetc. it is a matter of interpretation, as there are no rules the quarg outfit design could follow yet. one could argue that the crystal to convert energy in such a powerful laser is rather heavy, and that the first lightweigt look adds to the alieness of that weapon.
as others said, the design of your weapon is a quite “heavy“ approach in feel, but for my taste way to close to pug(yellow-reddish shades) or korath(rather bulky design), and may lack details (though, that always can be intentional)

**Comment by MCOfficer** [comment]
about the design, i don't think clunky outfits would fit for a species this advanced. although the image of the quarg station interior suggests otherwise, the quarg seem be advanced enough to make their technology not only effective, but also elegant:
`The Quarg city is a marvel of advanced technology, entirely enclosed within glass and steel` 

however, the other statements about them aren't exactly packed with information:

`strange colors, and walls that never quite seem to meet at right angles`
`In the docking section are flexible, oddly organic-looking tubes`

i personally have problems drawing any useful information out of this, so i'd follow the skylark design with the dark color and thin/elegant connection tubes.

**Comment by Tadrix** [comment]
> Indeed, precisely the original point of the design.

So the main idea is that only these three spikes rotate and everything else stands still, making up some sort of friction-less system?

**Comment by Brick63** [comment]
Yes

**Comment by KiLEdEnNis** [comment]
that would also mean a good turn rate

**Comment by accursedCursive** [comment]
Of course my design is bulky, it's for a 60 tonne laser.
I only spent like, an hour on it, I might redo it.
All of you are saying that the Quarg can make their technology lightweight as justification for why this design looks lightweight, but then shouldn't the outfit itself be lightweight?
Either the outfit should be lighter, or the appearance should be heavier.

The best solution in my opinion is to accept that the Skylance is a heavy weapon, then give Quarg light weaponry.
I've already given Quarg five more weapons in my mod:
-Energy Cannon (4x the damage of a regular blaster and 1.5x the velocity, slightly slower firerate)
-Capacitive Energy Cannon (Burst fire upgrade of above)
-Claw Energy Cannon (Short range, low accuracy rapid-fire energy cannon)
-Plasma Blade (Short range extreme-damage turret that thrusts you towards enemies with one plasma jet, and attacks with another)
-Negative Blade (Same as above but with a jet of negative energy, which creates negative recoil, meaning only the forward jet is necessary; it's better in every way except it's expensive and bad for damaging hulls)

**Comment by MCOfficer** [comment]
> but then shouldn't the outfit itself be lightweight

outfit space is justifiable, and mass... well technically it could be lored by compressed materials (e.g. the crystal could be very heavy) - still, that's a good point

**Comment by KiLEdEnNis** [comment]
> Of course my design is bulky, it's for a 60 tonne laser.
I only spent like, an hour on it, I might redo it.
All of you are saying that the Quarg can make their technology lightweight as justification for why this design looks lightweight, but then shouldn't the outfit itself be lightweight?
Either the outfit should be lighter, or the appearance should be heavier.

bla bla bla, I'll quote myself again:

> you're assuming the materials and whatever tech is in it can't be heavy, while it can
for example, maybe they used compression technology for the core

and what Wrzl said:

> one could argue that the crystal to convert energy in such a powerful laser is rather heavy, and that the first lightweigt look adds to the alieness of that weapon.


basically what this means is:
because the Quarg are such advanced aliens, we don't know much about their complex technology, this sprite has levitating parts, and they may use some advanced compression technology, meaning that the argument that such a heavy turret can't be that non-bulky, is invalid
adding that the turret is a bit wide in design, the outfit space can very well be 60 as well
if it makes ya happy, the Quarg will have a rework sometime, and that may include a rebalance of the  Skylance and other tech as well

**Comment by accursedCursive** [comment]
Everything is about the Quarg is just giving them ridiculous traits with zero explanation.
9600 capture defence on a ship at no outfit space cost? Alien magic.
An ultra-efficient battery? Alien magic.
A Firebird-sized ship that inherently generates more energy than three Armageddon Cores? Alien magic.
The Quarg are the Sonic OC of the ES galaxy. Sonic OCs are like "he can run faster than sonic and blow up the world but he chooses not to" and the Quarg are like "they have super powerful ships that are indestructible and have unlimited outfit space and energy and they could easily take over the galaxy but they choose not to".
Plus, the Quarg don't have anything that humans don't (except Jump Drives), their technology is just an upgraded subset of human technology.

The direction that I think Quarg should move in is to have technology that isn't as ridiculously optimised as it is now, and instead have much more exotic technology.
For example, instead of having massively overpowered energy generation, they have things like turrets that weave solar collectors out of electrons, giving them a huge but lightweight solar collector.
Instead of the Nanotech Battery being an upgraded battery, it's comparable to a similar-sized human/hai battery but also maintains the ship and recycles waste energy.
For their bulk power storage, if they need it, they can once again use turret-mounted technology.

Overall, the result is that the Quarg don't have anything that is like a human equivalent but several times better in the most obvious metric (Twice as good at best), but they have such a wide variety of technology at their disposal that they can build much more capable ships.

**Comment by Startome** [comment]
Actually quarg are T3.
Pug and Drak are destined T4 races.
They can crush the quarg fleet vs fleet.
Hardly the best magic

**Comment by jafdy** [comment]
I, for one, am looking forward to angular and alien looking quarg ships and outfits.

There is a famous quote from a si-fi writer that goes "technology sufficiently advanced is indistinguishable from magic", so "alien magic" seems quite appropriate.

**Comment by KiLEdEnNis** [comment]
your point isn't that invalid, but the Quarg are basically a placeholder by now until MZ creates more Quarg content and rebalances the current technology
creating sprites for the existing outfits is fine, they'll just likely change in stats, but apart from that we can't do much for the Quarg

**Comment by Tadrix** [comment]
@accursedCursive What exactly is your point? Are you…
- … trying to hijack this PR with your propositions? You should open your own PR instead.
- … criticizing this PR? Then you could be constructive and point out alternatives.

Currently you aren't providing alternatives, because I don't see your PR with new, alternative outfitter icons. What you are doing here is bragging about your mod, instead of offering help in creation of new content.

**Comment by accursedCursive** [comment]
I haven't done a pull request because it's a lot of work, and my content makes more sense when you see the bigger picture.
I'll release everything I've done as a mod eventually, and anyone can feel free to integrate content from it.

**Comment by KiLEdEnNis** [comment]
> I haven't done a pull request because it's a lot of work

surely someone of your caliber with several hours of C++ knowledge would be able to quickly make a good PR

**Comment by jafdy** [comment]
Even I can make a PR, with limited coding knowledge and limited artistic skill.

**Comment by accursedCursive** [comment]
@KiLEdEnNis 
I mean it's a lot of arguing to do a PR.
The people here argue with _everything_ I say, to the point that if I say A they'll say B then if I say B they'll say A.
Modding is 10x easier than contributing for this reason.

**Comment by Tadrix** [comment]
You haven't even tried yet.

Having your propositions challenged is a normal part of the process. and is meant to pass bullshit filter as well as ensure you've made your homework, didn't mess up on the way and that your additions will align with where the development is heading. There are many reasons your PR could be rejected, and not all of them are your fault. If everyone followed your logic, nothing would ever get done.

That said, nobody's forcing you to contribute anything, and you can live happily and mod away. Just don't brag about it here. This repo is for contributions and issues of _Endless Sky_ vanilla and official plugins. Come back when you have something to contribute or issues with vanilla game (and not one of your mods).

**Comment by KiLEdEnNis** [comment]
@Tadrix we have explained him on numerous occasions, I understand you try to help, but others agreed it wasn't worth arguing about anymore
keep up your involvement though, your post quality has improved a lot

**Comment by Brick63** [comment]
I would like to ask that all posts relate to the topic at hand and not be posted unless it is reasonably on topic

**Comment by KiLEdEnNis** [comment]
the Skylance description seems a bit over the top, if ya want I can rewrite that 1 & write initial descriptions for all outfits? (they can be fine-tuned later but it should at least provide a good basis)

**Comment by Brick63** [comment]
It was mainly a place holder until a design theme is decided upon

**Comment by KiLEdEnNis** [comment]
what exactly are we waiting for though? I think the design theme is good & so do some others, I haven't really heard of anyone that doesn't like it

**Comment by Lineth** [comment]
I'd like the idea of the Quarg weapons being 'modest damage-per-ton (for T3), butr very, very efficient', to match the idea of the Quarg being efficient in most of their tech (as per one of the Korath plasma thruster descriptions).

**Comment by Brick63** [comment]
well im going to assume that this design theme is good so ill start to work on the other outfits

**Comment by accursedCursive** [comment]
On the topic of Quarg outfits, how about something like this for the Graviton Thruster?
graviton thruster large
I've got the other 4 sizes (this is large, referred to as "Galaxy" in-game), and steering counterparts.
The detail is a bit simple, it's really just a suggestion for the general appearance of Graviton Thrusters.

**Comment by jafdy** [comment]
If you want to create your own design theme then go ahead, just don't do it in Brick's thread.

**Comment by Pizzavu** [comment]
Just created an account to say this and... AccursedCursive, you said that "the Quarg are like "they have super powerful ships that are indestructible and have unlimited outfit space and energy and they could easily take over the galaxy but they choose not to"." But there's actually a reason for that in lore? If I remember correctly, there's a much stronger alien race in-game that wouldn't allow them to take over the galaxy and they're very much aware of the fact. Their culture also is heavily based on peace, unless I'm mistaking them with the Hai (Haven't played for quite a bit)

**Comment by KiLEdEnNis** [comment]
it's nice that ya created an account and may contribute, but this is quite off-topic, especially as jafdy already stated that Accursed shouldn't post his stuff here

**Comment by Brick63** [comment]
the anti missile turret is now done

**Comment by Lineth** [comment]
When you finish a few Quarg outfits, did you plan on making new/remodeling the existing Quarg ships? Asymmetrical ships with this design/color scheme would look very, very cool.

**Comment by Brick63** [comment]
maybe another time

**Comment by Brick63** [comment]
the shield generator and battery are done

**Comment by Brick63** [comment]
added the engine sprites plus 2 other sizes

**Comment by Brick63** [comment]
some changes to the battery and added the Antimatter core
also no rush in answering @endless-sky but what are your thoughts on a cooling outfit for the quarg

**Comment by accursedCursive** [comment]
I don't suppose anyone cares, but I already have an entire collection of Quarg outfits and ships that dwarfs the current Quarg content, including sprites for everything you've made sprites for so far except the Nanotech Battery and the Antimatter Core (I deprecated AC because it didn't seem Quargy enough).
The detailing of the outfits isn't as good since I'm one person working on many outfits instead of many people working on few outfits; but the artstyle is a better match for Quarg, the lore and sci-fi is more interesting, and the resultant gameplay is far better.

Going by the number of images added to outfits and ships since it's the easiest way to count, the amount of ships and outfits I've added to Quarg exceeds the amount of ships and outfits added to the entire game by the 0.99 update, and the gameplay quality of it is exceptional since Quarg fleets now follow a completely different dynamic to Human fleets while still retaining a similar effectiveness:cost ratio to vanilla Quarg fleets.
The amount of time it took me to do that is less than the combined amount of time you people spend in one day arguing about what to do without actually doing it.

Brick should be given a medal for actually doing something and doing it well, except that I still hold the opinion that the artstyle is totally wrong for Quarg. It'd be great for Archon though.

**Comment by MessyMix** [comment]
@accursedCursive.

That's funny.

**Comment by accursedCursive** [comment]
Also, do you have a sound for Graviton Engines yet?
I made a pretty good one by taking the longest echo I could find in a certain song, cutting off the high frequencies with an equaliser, adding an echo, playing a reversal of the same sound at the same time, then slowing the result down to half speed.
[Graviton Engines.zip](https://github.com/endless-sky/endless-sky/files/1662603/Graviton.Engines.zip)

**Comment by tehhowch** [comment]
@accursedCursive you are more than welcome to open a PR with your content. Please ensure that it meets the game's [Quality Checklist](https://github.com/endless-sky/endless-sky/wiki/QualityChecklist) and provides proper documentation of all utilized assets. Distribution of copyrighted / protected material outside the bounds of its license is not allowed and will never be tolerated in this (or hopefully any) repo.
A web search or two should provide more than sufficient documentation on how to create the pull (forking the repo, creating a branch for your content, adding your new content, pushing the commits to the web, and finally opening the request).

@Brick63 I'd agree that the Quarg will eventually be in need of a cooling outfit. Same for a hull repair outfit.
Given that the game is still developing the first T2 story, I'm not sure it is known exactly what outfits are going to be needed, since the outfit diversity will stem from the ship diversity, and ship diversity will stem from the depth of the story. We could make assumptions, based on the naming of the engine - "Medium Graviton" - that there will be 3 or more of each, and prepare 3 sizes of "utility" outfits like engines and core systems (batteries, hull/shield, cooling, energy). Or if not actually making all 3, choosing a design that allows simple extension to the other sizes.

**Comment by accursedCursive** [comment]
My content has to stay separate from vanilla content.
The balancing rules of my content are completely different; better, but it's a mess when it interacts with unmodified vanilla content.

A ship from my mod has low health, but high damage and agility.
A vanilla ship has high health, but low damage and agility.
A ship from my mod fighting an unmodded vanilla ship results in a pretty skewed fight, since the ships in my mod typically trade off DPS for other traits such agility or being able to hit fast targets more easily.
A vanilla ship using outfits from my mod would have very high health, high damage, and high agility.
A ship from my mod using vanilla outfits would have extremely low health, low damage, and low agility.

I do wish a total balance overhaul would happen sometime. I'm serious when I say it's way better to balance the game the way I do. Currently, balance works like this:
-Expensive ships have a low damage:cost ratio but you won't lose them in every fight
-Cheap ships have a high damage:cost ratio but you will lose them in every fight
There's nothing more to say. All weapons are more or less the same, and an interceptor is just a much weaker cruiser because to hit anything that interceptor has to get so close that it has no chance of dodging anything.
It's not fun, I haven't touched vanilla ES in about a year.

**Comment by jafdy** [comment]
That nice . . .
WHY IS IT HERE??

**Comment by Startome** [comment]
Seriously? This is Brick's post, for Brick's work. Here you comment on helping with his work. In this post, nobody cares abou your work. Go make your own post, for your work, and there, in your separate post from this one, folks will care about your work. Stop shamelessly using other people's posts to promote your stuff, it's becoming spam.
Go make your own post. And take this stuff there.

**Comment by kikotheexile** [comment]
"My mod is better and i dont play vanilla" helps no one. Stop trolling.

**Comment by accursedCursive** [comment]
I do make my own threads, but on the Steam forums Kiko locks most of the threads I make (they lock any thread that doesn't agree with their opinion) and MZ is a nuisance here.
Community-wise, this game is totally messed up.
The game relies on modders, but the community will defend existing content to the death. No alterations allowed, only new content, as if zero mistakes can be made by an amateur game developer building a game out of contributions from random strangers.
My modding alters pretty much everything, and my suggestions recommend it as well, so thanks to anti-modding moderators like Kiko and MZ I'm not allowed to have my own threads.

**Comment by Brick63** [comment]
Please don't post this stuff on this thread it is off topic and detracts from the conversation so take this somewhere else

**Comment by accursedCursive** [comment]
Oh come on, don't you start whining as well.
You're the one other person in the ES community who spends more time making content than whining and/or circlejerking, though I don't use the ES Discord so I can't be sure.
Also, why do you faithfully make content according to poor guidelines? Antimatter Core isn't even used in the game and frankly never should be (at least not by Quarg), yet you still made a model of it. You might as well make a few outfits of your own, the Quarg are in sore need of more outfit diversity. And Pug. A few Korath outfits wouldn't be amiss either, you could do what I do and make it so that Quarg have their own versions of certain Korath outfits given that it makes sense with ES lore (unless you adhere to the whole tier system nonsense some users follow and thus assume that Quarg cannot make Quarg-Korath hybrid technology that provides an effective sidegrade to conventional Quarg technology).

**Comment by Amazinite** [comment]
If your problem is with Kiko then don't make it our problem as well.

**Comment by Brick63** [comment]
im not whining simply asking that this thread stay on the topic that is at hand, no complaining about people, no talks about balance, no talk of anything past the subject of this thread and i would like EVERYONE to refrain from anything other than the subject at hand.

**Comment by Brick63** [comment]
the countermeasure is done and so are all other current outfits

**Comment by Brick63** [comment]
well with all the sprites done im open to suggestions for descriptions

**Comment by KiLEdEnNis** [comment]
I'm personally not sure about the engines but the rest look beautiful

now I think it's just waiting for MZ's review (and confirmation on a Quarg cooling & hull repair outfit, unless ya wanna start on them anyway)

ya could also try & make more Quarg ships but that'd require some coordination with MZ (like what kinda ships does he want, how should they generally look etc.)

**Comment by Brick63** [comment]
screenshot 2018-01-25 16 07 03
here is a picture of how the hard-points look in game

File: data/quarg outfits.txt [hunk 4]
```diff
@@ -102,7 +102,7 @@ outfit "Intrusion Countermeasures"
 outfit "Medium Graviton Thruster"
 	category "Engines"
 	"cost" 20000000
-	thumbnail "outfit/medium quarg thruster"
+	thumbnail "outfit/medium quarg thrusters"
```
Comment (KiLEdEnNis) [inline data/quarg outfits.txt]: the small & large ones are "[small/large] quarg thruster" so maybe change this back to thruster & rename the "medium quarg thrusters" so singular for consistency?

**Comment by Brick63** [comment]
the assets are now up

**Comment by Brick63** [comment]
fixed so that it is now up to date with the new hull repair

**Comment by accursedCursive** [comment]
I still think the dark grey and blue looks totally out of place on shiny, bright-coloured Quarg ships.
skylance

**Comment by MessyMix** [comment]
@accursedCursive I actually like that sprite more than the current proposal, but 1. the skylance shoots blue anyways, and 2. one of the two quarg ships are dark in color, so I don't see how it's "totally" out of place.

Your color scheme I personally like more, but I don't know if it actually makes more sense than the current one.

**Comment by jafdy** [comment]
The green is nice, but I think the dark colour fits the skylark and the effects better (I also feel like skylark is a more quarg like ship than the wardragon).

**Comment by MCOfficer** [comment]
it fits the skylark very well, but looks odd on the wardragon..

**Comment by Wrzlprnft** [comment]
I like the greenish touch, too.

**Comment by accursedCursive** [comment]
Personally I think that the Quarg aesthetic should be robust and simple, yet creative and effective technology.
I have my own version of Quarg lore which explains why but it'd totally derail this thread.

In short, they don't want to make fancy stuff because that creates a dependence on ultra-high-tech industries, which are expensive and irreplacable. The Quarg have outlasted many other races, partially because by developing less advanced technology they can afford to suffer some industry damage and afford to lose a few ships. The other part is because instead of sprawling across a hundred systems they just put their entire population in only a few systems, which prevents the empire from fragmenting.

Headcanon may be a dumb reason for disliking the look of these outfits, but I really do think that uniform and fancy-looking technology would better suit a race that will doubtlessly be more advanced than the Quarg. I really like the idea of an exotic range of more rugged-looking outfits being available to Quarg.

**Comment by Tadrix** [comment]
@accursedCursive Again, please, take it to Google Groups (since you say you are no longer welcome in Steam Community).

Brick is making images for existing outfits. Making new outfits is outside the scope if this issue. And he isn't whining, he is just pointing this out to you.

> I already have an entire collection of Quarg outfits and ships that dwarfs the current Quarg content

Dear ~Quarg Mary Sue~ accursed One,
Regarding your difficulties with Kiko and others: you might have a problem. This community usually is responsive and reasonably helpful. To some it would seem even friendly, but it could be you aren't any longer one of those people. The reasons include but are not limited to:
- Your attitude (you behave as if everyone owed you anything), which makes people unwilling to have anything to do with you;
- Your inability to back off when presented with a compelling argument against your proposals;
- Your disregard for the topic at hand (the topic in question being a pull request for a vanilla game, or an issue with said game): if you feel as if you can't post and stay on topic, you probably shouldn't post on GitHub; and finally
- Your inability to contribute to Endless Sky in any way (with issue reports or any additions).

Both of the last two points stem from the same issue: You've came to the wrong repository, because you are playing an entirely different game than the rest of us (GitHub is for PRs and issues concerning vanilla game, everything other shouldn't be here). If you've really come to contribute to _Endless Sky_ and not _Twilight of the Quarg_ or whatever your mod is called, then you should play the vanilla game through, get better, understand the game as it is and only then propose any alterations. Until then we have really nothing to discuss.

**Comment by accursedCursive** [comment]
Organic ships with inorganic outfits make a lot of sense, the Archon would be pretty cool like that.
What are they going to do with organic outfits, shoot worms at enemy ships?
I do like the idea of a pure-organic race, but it shouldn't be the Archon or Pug.
Besides, check out how well Brick's outfits suit Archon:

capture

**Comment by MCOfficer** [comment]
the hardpoints fit the archon well (at least better than the wardragon). however, as much as i like the idea, archon outfits seem to be magical - meaning there's no beam or projectile, but the outfits just affect whatever they want, no matter the circumstances. also the archons supposedly being [images? depictions?] of the 4-dimensional drak in our 3-dimensional world might suggest that archons don't really have such things as outfits, but just **function**, similar to a hoverboat not having wings (yes, that's an awful comparison).

File: copyright [hunk 5]
```diff
@@ -277,6 +277,16 @@ Copyright: Various
...
+ 
+ Files:
+ images/outfit/quarg*
+ images/hardpoint/quarg*
+ images/outfit/small?quarg*
+ images/outfit/medium?quarg*
+ images/outfit/large?quarg*
+Copyright: Evan Fluharty (Evanfluharty@gmail.com)
+License: CC-BY-SA-4.0
+Comment: Derived and completed from works by Maximilian Korber (Wrzlprnft), Fernando S. Sáez (Azure Nyoki), and originally drawn up by Tommy Thach (Bladewood) (all from under the same license)
```
Comment (tehhowch) [inline copyright]: I know you operated from preliminary work from the named contributors here when developing the skylance sprite, but is that also true for the rest of the quarg outfits? If not, I imagine you are able to attribute sole copyright to those images.

i.e.
```
Files:
  images/outfit/quarg?skylance
  images/hardpoint/quarg?skylance
Copyright...
License..
Comment: Derived and completed from works by Maximilian Korber (Wrzlprnft), Fernando S. Sáez (Azure Nyoki), and originally drawn up by Tommy Thach (Bladewood) (all from under the same license)

Files:
  images/outfit/quarg*
  ...
Copyright... 
License...
```

**Comment by accursedCursive** [comment]
The game describes Archons as such:
> Archons are partly biological, partly technological, and partly something that human language has no words for. They are extremely powerful. The Archons are somehow affiliated with the Drak, yet separate from them.
Thus, it's perfectly appropriate to have their turrets be physical.

File: copyright (hunk 5 above)
Comment (Brick63) [reply copyright]: I'll adjust it more later

**Comment by Tadrix** [comment]
An answer perfectly suitable for your argument: Sentient, biological beings (including humans) tend to use guns instead of carrying turrets on their backs. However, if you are a mage, some of your spells could function similar to turrets. And the game uses turret slots for Archons only because it's suitable for weapons which shoot in any direction instead of forward. Players aren't supposed to be able to use these "turrets".

**Comment by KiLEdEnNis** [comment]
these outfits are very fancy alien devices, suitable for the Quarg
I think an Archon's weapons should be more half-organic complex-shaped outfits

**Comment by accursedCursive** [comment]
I took the colour scheme of a Wardragon and applied it to a turret.
skylance
I'm using Brick's sprites as Archon outfits and my own sprites as Quarg outfits, and it works pretty well except that I'm using the Cycles renderer and I don't know the lighting angle all the other turrets use.

**Comment by KiLEdEnNis** [comment]
> Please don't post this stuff on this thread it is off topic and detracts from the conversation so take this somewhere else

Brick 3 days ago

**Comment by Brick63** [comment]
im fine with the posting of different ideas for the outfits but at this point it would have to be the full set of them with a very nice quality, also that turret design looks nice and could be great with a little more detail.

File: copyright [hunk 6]
```diff
@@ -277,6 +277,22 @@ Copyright: Various
 License: public-domain
  Taken from unsplash.com, a collection of photographs that have been donated and
  placed in the public domain.
+ 
+ Files:
+ images/outfit/quarg*skylance*
+ images/hardpoint/quarg*skylance*
```
Comment (Amazinite) [inline copyright]: `quarg?skylance*`

Comment (Brick63) [reply copyright]: oops

**Comment by accursedCursive** [comment]
@Brick63 
If you want to assign these sprites to Quarg outfits, you should only do so if you intend for them to be the final design for Quarg outfits and not later be pushed to another race like the Archon, since the Endless Sky community adds content much more readily than it moves it around.
If this content isn't perfect for Quarg, it won't matter; it's probably going to stay there forever.
Consider the Skylark and Wardragon for example: Whichever came second, it completely mismatched the theme of the first, but stuck anyway.

My view is that the Quarg have had no outfit sprites for ages, and they can wait a little while longer if it's necessary to make sure that each race gets aesthetically-appropriate outfits.
With the exception of the Graviton Engines (which is a weird blend of both our art styles so it doesn't suit any race), what you've made suits the Archon theme best.

Quarg, however, need a colour palette more like this:
untitled
The four colours on the left are intended to be used in large amounts, the four colours on the right are for detailings.
This is based on the Wardragon, so the colours are vivid, and to exaggerate that property there's a good mix of bright colours for outfits to choose from.

**Comment by Wrzlprnft** [comment]
but skylark and wardragon both predate 0.7.8 and with that any github version. 

and bricks color palette seems to be more skylark oriented anyway.

**Comment by accursedCursive** [comment]
But the Skylark colouration is far worse than that of the Wardragon.
It's a black ship on a black background, with no colour except for a hint of purple.
Any turrets will firmly contrast against the ship, and there are no sensible hardpoints defined for the ship.
Future Quarg ships will not look like the Skylark; the only way to recognise a moving Skylark is that it's a dark smudge with a few turrets, so if there were five more ships that also looked like a dark smudge with turrets while moving, adding more ships would have been for nothing.

**Comment by KiLEdEnNis** [comment]
it's not hard to see a Skylark at all, I easily spotted one as seen below
temp12

**Comment by accursedCursive** [comment]
It turns out that it's extremely difficult for a Skylark to look like a Wardragon but not look like a Wanderer ship.
skylark

**Comment by jafdy** [comment]
And what about making the wardragon dark. Why should we make the skylark so bright.

**Comment by accursedCursive** [comment]
All the new ships of late have looked really dull.
Really detailed, but it's always just shades of grey..
It'd be nice to have some new colourful ships for once.

**Comment by Amazinite** [comment]
>All the new ships of late have looked really dull.

The Coalition and Remnant ships have been dull (which is just a matter of the design style of those factions), but the newest ships to be added are looking pretty nice.

image
image

**Comment by accursedCursive** [comment]
I think I have the hang of outfit rendering now.
skylance
I'm still using ambient lighting though because hard shadows make this look awful.

**Comment by Disiuze** [comment]
@accursedCursive That's nice, however, I would like you to know that it looks like you're hijacking the PR. Please stop.

**Comment by Wrzlprnft** [comment]
Yup. that is just straight up an alternative. How about just opening an alternative PR?

**Comment by Pointedstick** [comment]
Overall these are very nice, though I'm not sure about the thruster images. They looks a bit too simplistic for my tastes, and the purple glow doesn't match the color scheme uses for the other images.

**Comment by Lineth** [comment]
I genuinely love the engines. And all the sprites and artwork you've done for the Quarg. Makes me genuinely wish for a Quarg campaign. I'd love to see some ship concepts as well. No doubt they'd be killer.

**Comment by Brick63** [comment]
i tried the ships and they didn't turn out as well as i had hoped, however in #3665 the ships were done by someone and i love the designs he did. Doing these engines was much more enjoyable than the previous ones and look much better.
